src/services/api/me/me.service.ts

```diff
@@ -93,10 +97,35 @@ export class MeService {
         );
         continue;
       }
-      mySpaceResults.push({
-        space: activityLog.space,
-        latestActivity: activityLog,
-      });
+      if (!showOnlyMyCreatedSpaces) {
+        mySpaceResults.push({
+          space: activityLog.space,
+          latestActivity: activityLog,
+        });
+      } else {
+        if (activityLog?.space) {
```

Comment:
this check seems redundant

Reply:
It is not, because activities do not have reference to spaces, and when a space is deleted, this is being hit.